debian/xenial/kubelet/lib/systemd/system/kubelet.service

```diff
@@ -1,10 +1,12 @@
 [Unit]
-Description=Kubernetes Kubelet Server
-Documentation=https://github.com/kubernetes/kubernetes
+Description=kubelet: The Kubernetes Node Agent
+Documentation=http://kubernetes.io/docs/
 
 [Service]
-ExecStart=/var/lib/kubelet/kubelet-wrapper
+ExecStart=/usr/bin/kubelet
```

Comment:
Ok, so the user who wants to set any flags would have to create a drop-in like `/etc/systemd/system/kubelet.service.d/99-override.conf` containing:
```
[Service]
ExecStart=
ExecStart=/usr/bin/kubelet --foo --bar
```

They get the freedom of adding anything there, without depending on whether the variable is called `$KUBELET_OPTS` or `$KUBELET_ARGS` and how that gets expanded etc. If they are using config management of sorts, they don't have to put append lines to a shell script and care about what lines were there before, instead they simply write the drop-in unit.